docs/reference/offline-stores/remote-offline-store.md

```diff
@@ -25,4 +27,7 @@ The complete example can be find under [remote-offline-store-example](../../../e
 
 ## How to configure the server
 
-Please see the detail how to configure offline feature server [offline-feature-server.md](../feature-servers/offline-feature-server.md)
\ No newline at end of file
+Please see the detail how to configure offline feature server [offline-feature-server.md](../feature-servers/offline-feature-server.md)
+
+## How to configure Authentication and Authorization
+Please refer the [page](./../../../docs/getting-started/concepts/permission.md) for more details on how to configure authentication and authorization.
```

Comment:
do we need to mention same lines here or reference of permission doc (https://github.com/RHEcosystemAppEng/feast/blob/feast-rbac/docs/reference/feature-servers/python-feature-server.md) 

https://github.com/RHEcosystemAppEng/feast/blob/feast-rbac/docs/reference/feature-servers/offline-feature-server.md#how-to-configure-authentication-and-authorization-